.github/CONTRIBUTING.md

```diff
@@ -71,22 +72,44 @@ Documentation for running and updating our existing tests, as well as the code f
 
-Otherwise, if you're interested in backporting a change from WordPress, then please explain in a pull request or issue what the specific change is, how it works, and why it's something that makes sense for ClassicPress to adopt. Often there is a lot of good discussion about each change in the relevant WordPress Trac tickets, and it makes the maintainers' job much easier to see this discussion summarized and linked.
+### Tips for a good Backport PR.
+- Give the PRs a clear title. Explain in a pull request or issue what the specific change is, how it works and why ClassicPress in particular should adopt it. Often there is a lot of good discussion about each change in the relevant WordPress Trac tickets. It makes the maintainers' job much easier to see this discussion summarized and linked.
+- Explain what testing has been done on the PR and what may be left to do. Screenshots, gif, video and text instructions documenting the tests can be very useful. Those tell other contributors looking at the PR how to verify the changes. "This was tested in WordPress" **is not enough** to meet our review criteria, see [Review Criteria](#review-criteria) above.
+- Look to make sure the backport is done correctly, with attribution to the original WP committer. 
+- If there are merge conflicts during the backport, check if they have been resolved correctly by comparing the final changes from the full PR against the corresponding WP changeset(s). Remember to fix these **in a separate commit**. If you cannot resolve the conflicts, ask for help in the [Core slack channel](https://www.classicpress.net/join-slack/) or just push your branch as-is and we'll take care of it!
 
-If you're not sure about any of that, then it's a good idea to ask first. A good way is to create an issue for the specific change you're interested in, along with links to the relevant WordPress changesets and tickets, and any other information you have about how the change works.
+### Making a Backport PR.
+When you're ready to backport a code change, make a new branch to your ClassicPress fork:
+
+- Identify the WordPress **changeset number** that you'd like to port such as `43123`.
+- Run `bin/backport-wp-commit.sh` script in your terminal/command prompt.
+
+```
+bin/backport-wp-commit.sh "changeset-number"
+```
+ to apply the change to the code. 
+ 
+ If you're porting multiple changesets, you can use the `-c` option to this script 
+
+```
+bin/backport-wp-commit.sh -c "changeset-number"
+```
+to apply all the changesets to the same branch. 
 
-Finally, when you're ready to backport a code change, identify the WordPress **changeset number** that you'd like to port, and run the `bin/backport-wp-commit.sh` script to apply the change to the code. If you're porting multiple changesets, you can use the `-c` option to this script to apply all the changesets to the same branch.
+Then push your branch for review as a PR.
```

Comment:
Say how to push your branch. At least mention it's your fork, not the ClassicPress repo.

Reply:
Agree this might have to change with the new section of making your local dev environment. Let's fix them hand in hand.